.dockerignore

```diff
@@ -0,0 +1,22 @@
+# Build
+/_build/
+/cover/
+/deps/
+/docs/
+/.fetch
+*.ez
+node_modules
+
+# Erlang
+erl_crash.dump
+
+# Public
+/public/uploads
+!/public/uploads/test.txt
+
+# Dev
+/.git
+/.github
+.env
+.DS_Store
+Thumbs.db
```

Comment:
Should we also ignore `.elixir_ls`? (generated by a popular VSCode plugin for Elixir, some of us are using VSCode now)